sdk/simulator-service/src/implementation/strategies/Refinance.ts

```diff
@@ -119,12 +110,15 @@ export async function refinace(
     .next(async () => ({
       name: 'CollateralSwap',
       type: SimulationSteps.Swap,
-      inputs: await dependecies.getQuote({
-        from: args.position.collateralAmount,
-        to: args.targetPool.collateralTokens[0],
+      inputs: {
+        ...(await dependecies.swapManager.getSwapQuoteExactInput({
+          chainInfo: args.position.pool.protocol.chainInfo,
+          fromAmount: args.position.collateralAmount,
+          toToken: args.targetPool.collateralTokens[0],
+        })),
         slippage: args.slippage,
-        fee: 0,
-      }),
+        fee: dependecies.getSummerFee(),
```

Comment:
Right now it might be fine, but in future we might need to add some parameters to this dependency.

Reply:
feel free to do it it's a placeholder as you see :)